only supports being run on "[pull_request]"

Thanks for the cool action!

I tried adding it on my existing push-triggered workflow, and got a confusing error on my first try:
```
Run preactjs/compressed-size-action@v1
  with:
    repo-token: ***
    build-script: build
    show-total: true
    collapse-unchanged: true
##[error]Cannot read property 'base' of undefined
```

Looking into the code, it appears to be because the action depends on `context.payload.pull_request`: https://github.com/preactjs/compressed-size-action/blob/6cd4d74bf28d472327aeebe8e71dbc903a85da7f/action.js#L23
which presumably doesn't exist on push-triggered workflows.

I wanted the action run on push because a PR can go through multiple iterations and I wanted the compressed size reported for every iteration.

Ideal would be to support both types of triggers (it appears there is some [commented out code](https://github.com/preactjs/compressed-size-action/blob/6cd4d74bf28d472327aeebe8e71dbc903a85da7f/action.js#L22) to use the octokit API to fetch PR data), but at the very least it would be nice to fail a bit more gracefully (with a clearer error message).

--- a/action.js
+++ b/action.js
@@ -24,6 +24,9 @@ async function run(octokit, context) {
 	try {
 		debug('pr' + JSON.stringify(pr, null, 2));
 	} catch (e) {}
+	if (!pr) {
+		throw Error('Could not retrieve PR information. Only "pull_request" triggered workflows are currently supported.');
+	}
 
 	const plugin = new SizePlugin({
 		compression: getInput('compression'),

